static/src/javascripts/projects/commercial/modules/creatives/fabric-expandable-video-v2.js

```diff
@@ -32,17 +32,15 @@ define([
 
         function create() {
             var videoHeight = openedHeight;
+            var plusIconPosition = params.showCrossInContainer.substring(3);
```

Comment:
some stupid convention in the DFP template, strings will come as `To something` and I need the `something` part.